todo-react/src/pages/Home.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { Todo } from "todo";
+import Card from "../components/Card";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  background-color: #f5f5f5;
+`;
+
+const AppHeader = styled.header`
+  width: 100%;
+  padding: 1rem;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: #2c3e50;
+  color: #fff;
+  margin-bottom: 1.5rem;
+`;
+
+const TodosContainer = styled.ul`
+  min-width: 300px;
+`;
+
+const AddTodoForm = styled.form`
+  min-width: 300px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  display: flex;
+  background-color: #fff;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  margin-bottom: 1rem;
+  transition: all 0.2s ease-in-out;
+`;
+
+const AddTodoInput = styled.input.attrs({ type: "text" })`
+  width: 100%;
+  font-size: 1rem;
+  font-weight: 400;
+  border: none;
+  outline: none;
+  background: transparent;
+  ::placeholder {
+    color: #2c3e506d;
+  }
+`;
+
+const Home = () => {
+  const [newTodoTitle, setNewTodoTitle] = useState<string>("");
```

Comment:
```suggestion
  const [newTodoTitle, setNewTodoTitle] = useState("");
```
이럴 때는 타입을 명시하지 않아도 자동으로 추론된답니다!